test/metabase/lib/equality_test.cljc

```diff
@@ -610,8 +610,8 @@
           ca-expr    (m/find-first #(= (:name %) "CA") columns)]
       (testing "different columns"
         (is (not= created-at ca-expr))
-        (testing "but both have the ID"
-          (is (= (:id created-at) (:id ca-expr)))))
+        #_(testing "but both have the ID"
+            (is (= (:id created-at) (:id ca-expr)))))
```

Comment:
@bshepherdson, should we keep the test and adjust the description?

Reply:
Yes, I think we should adjust the tests to enforce the behaviour we intend. In this case, that's having no `:id` on the expression.